lib/taskana-core-test/src/test/java/acceptance/classification/update/UpdateClassificationAccTest.java

```diff
@@ -0,0 +1,834 @@
+      // run the TaskRefreshJob that was scheduled by the ClassificationChangedJob.
+      runner.runJobs();
+      Thread.sleep(
+          10); // otherwise the next runJobs call intermittently doesn't find the Job created
+      // by the previous step (it searches with DueDate < CurrentTime)
+      runner.runJobs();
+    }
+
+    private void validateTaskProperties(
+        Instant before,
+        List<String> tasksUpdated,
+        TaskService taskService,
+        WorkingDaysToDaysConverter converter,
+        int serviceLevel,
+        int priority)
+        throws Exception {
+      for (String taskId : tasksUpdated) {
+        Task task = taskService.getTask(taskId);
+
+        Instant expDue =
+            converter.addWorkingDaysToInstant(task.getPlanned(), Duration.ofDays(serviceLevel));
+        assertThat(task.getModified())
+            .describedAs("Task " + task.getId() + " has not been refreshed.")
+            .isAfter(before);
+        assertThat(task.getDue()).isEqualTo(expDue);
+        assertThat(task.getPriority()).isEqualTo(priority);
+      }
+    }
+  }
+
+  @TestInstance(Lifecycle.PER_CLASS)
+  @Nested
+  class UpdateClassificationExceptionTest {
+    /**
+     * This BeforeAll method is needed for this {@linkplain
+     * #should_ThrowException_When_UserIsNotAuthorized test} and {@linkplain
+     * #should_ThrowException_When_UserRoleIsNotAdminOrBusinessAdmin test} since it can't create an
+     * own classification.
+     *
+     * @throws Exception for errors in the building or reading process of entities.
+     */
+    @WithAccessId(user = "businessadmin")
+    @BeforeAll
+    void createClassifications() throws Exception {
+      defaultTestClassification()
+          .key("BeforeAllClassification")
+          .buildAndStore(classificationService);
+    }
+
+    /**
+     * The classification used here is created in the {@linkplain #createClassifications method} in
+     * the beginning.
+     *
+     * @throws Exception for errors in the updating or reading process of entities.
+     */
+    @Test
+    void should_ThrowException_When_UserIsNotAuthorized() throws Exception {
+      Classification classification =
+          classificationService.getClassification("BeforeAllClassification", "DOMAIN_A");
+      classification.setCustomField(ClassificationCustomField.CUSTOM_1, "newCustom1");
+
+      MismatchedRoleException expectedException =
+          new MismatchedRoleException(
+              currentUserContext.getUserid(), TaskanaRole.BUSINESS_ADMIN, TaskanaRole.ADMIN);
+      assertThatThrownBy(() -> classificationService.updateClassification(classification))
+          .usingRecursiveComparison()
+          .isEqualTo(expectedException);
+    }
+
+    /**
+     * The classification used here is created in the {@linkplain #createClassifications method} in
+     * the beginning.
+     *
+     * @throws Exception for errors in the updating or reading process of entities.
```

Comment:
Do we need this comment?

Reply:
I think it's a good to know. Especially, when a new contributor is going through the code, he/she doesn't need to search where or when it was created.

Reply:
done :)